docs/development/release.rst

```diff
@@ -73,9 +73,9 @@ Steps for the day of the release:
 #. Check that the tarball and description (which is from ``LONG_DESCRIPTION.rst``) on PyPI is OK.
 #. Update the Gammapy stable branch to point to the new tag
    as described `here <http://docs.astropy.org/en/latest/development/releasing.html>`__.
-#. Add the new version on readthedocs: https://readthedocs.org/dashboard/gammapy/versions/
-   This will automatically trigger a build for that version, which you can check here:
-   https://readthedocs.org/projects/gammapy/builds/
+#. Add the new version of docs in Gammapy-web Github repository
```

Comment:
Put the exact repo name `gammapy-docs` and maybe even link to it here.

Reply:
Ok.
Done.